CSS/products.css

```diff
@@ -107,7 +110,7 @@ label > input[type="radio"] + *::before {
 	display: inline-block;
 	vertical-align: bottom;
 	width: 20px;
-	height: 2px;
+	height: 20px;
```

Comment:
use rem

Reply:
I want the radio button to be of specific size only . That is why px is being used here.